docs/examples/audio_decoder.ipynb

```diff
@@ -0,0 +1,238 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Audio Decoder in DALI\n",
+    "\n",
+    "This tutorial presents, how to set up simple pipeline, that loads and decodes audio data in DALI. Here we will use simple example from Speech Commands Data Set. While this dataset consists of samples in .wav format, following procedure can be used for most of the well-known digital audio coding formats.\n",
+    "\n",
+    "## Step-by-step guide\n",
+    "1. Let's start by importing handful of utils and DALI itself."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nvidia.dali.pipeline import Pipeline\n",
+    "import nvidia.dali.ops as ops            \n",
+    "import nvidia.dali.types as types\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "batch_size = 1\n",
+    "audio_files = \"audio\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "used `batch_size` is `1`, to keep things simple.\n",
+    "\n",
+    "2. Next, let's implement the pipeline. Firstly, you need to load data from disk (or any other place). Used FileReader is able to load data, as well as it's labels. For more information, refer to FileReader docs. Furthermore, similarly to image data, you can use Reader ops specific for given dataset or dataset format (see [CaffeReader](https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/supported_ops.html#nvidia.dali.ops.CaffeReader)). After loading input, pipeline decodes the AudioData. As stated above, the AudioDecoder operator is able to decode most of well-known audio formats.\n",
+    "   \n",
+    "   Note: Please remember, that you shall pass proper `dtype` argument to the operator. Supported types can be found in the documentation. If you have 24-bit audio data and you set `dtype=INT16`, it will result in loosing some information from the samples. The default `dtype` for this operator is `INT16`"
```

Comment:
```suggestion
    "   Note: Please remember that you shall pass proper data type (argument `dtype`) to the operator. Supported data types can be found in the documentation. If you have 24-bit audio data and you set `dtype=INT16`, it will result in loosing some information from the samples. The default `dtype` for this operator is `INT16`"
```